pilot/pkg/networking/plugin/authz/authorization.go

```diff
@@ -108,13 +116,15 @@ func buildFilter(in *plugin.InputParams, mutable *networking.MutableObjects) {
 						mutable.FilterChains[cnum].HTTP = append(mutable.FilterChains[cnum].HTTP, httpFilter)
 					}
 				} else {
+					tcpFiltersOnce.Do(func() { tcpFilters = b.BuildTCP() })
```

Comment:
is there a reason that tcpFiltersOnce are used. Do you mean the b.BuildTCP() should only be invoke one time?

following the function explanation here: https://golang.org/src/sync/once.go?s=1473:1500#L30